networking/managing-networking.adoc

```diff
@@ -0,0 +1,28 @@
+// Assembly filename: managing-networking.adoc
+// Explains network management
+[id='managing-networking’]
+= Managing Networking
+include::modules/common-attributes.adoc[]
+:context: managing-networking
+toc::[]
+
+This topic describes the management of the overall cluster network,
+including project isolation and outbound traffic control.
+
+// Managing the network when Multitenant is enabled
+include::modules/nw-management-multitenant.adoc[leveloffset=+1]
+//
+include::modules/nw-isolation-multitenant.adoc[leveloffset=+2]
+//
+include::modules/nw-joining-project-networks.adoc[leveloffset=+2]
+//
+include::modules/nw-multicast-multitenant.adoc[leveloffset=+2]
+include::modules/nw-enabling-multicast.adoc[leveloffset=+2]
+//
+include::modules/nw-project-networks-global.adoc[leveloffset=+2]
+//
+include::modules/nw-static-ips-ext-proj-traffic.adoc[leveloffset=+2]
+include::modules/nw-egress-ips-namespaces.adoc[leveloffset=+2]
+//
+// Managing the network when NetworkPolicy is enabled
```

Comment:
Put this module in a separate assembly. It's a different user story.

Reply:
Applied in https://github.com/openshift/openshift-docs/pull/14343.